setup.cfg

```diff
@@ -0,0 +1,34 @@
+##########################
+# Setup.py Configuration #
+##########################
+[metadata]
+name = metabolomics_spectrum_resolver
+version = 0.0.1-dev
+long_description = file: README.md
+long_description_content_type = text/markdown
+
```

Comment:
feel free to add appropriate author metadata here (and other package metadata). In some of my packages, this looks like
https://github.com/pybel/pybel/blob/b9171a2acbe7e858e9f2c809b4294d72fdee3cbe/setup.cfg#L18-L22